src/CountryHandler/Indonesia.php

```
@@ -0,0 +1,126 @@
+        }
+
+        // First 2 digits are the tax office code
+        $taxOfficeCode = substr($normalizedTin, 0, 2);
+
+        // Tax office code should not be 00
+        if ('00' === $taxOfficeCode) {
+            return false;
+        }
+
+        // Digits 10-12 are the branch code (KPP)
+        $branchCode = substr($normalizedTin, 9, 3);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$branchCode'. (undefined)

(UnusedLocalVariable)